website/docs/r/vpc_peering_options.html.markdown

```diff
@@ -0,0 +1,166 @@
+---
+layout: "aws"
+page_title: "AWS: aws_vpc_peering_connection_options"
+sidebar_current: "docs-aws-resource-vpc-peering-options"
+description: |-
+  Provides a resource to manage VPC peering connection options.
+---
+
+# aws_vpc_peering_connection_options
+
+Provides a resource to manage VPC peering connection options.
```

Comment:
It might be helpful to mention the ordering problem here to explain why you'd want to use this resource over the arguments in the original resources. Same vice-versa.

Reply:
Agreed.
I'll use text from this PR's description:

> Using a VPC Peering Connection Options resource decouples management of the connection options from
> management of the VPC Peering Connection and allows options to be set correctly in cross-account scenarios.